src/tests/test_featurize.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+from .. import featurize
+
+def test_placeholder():
+    data = np.random.rand(200).reshape(50,4)
+
+    for i in range(10):
+        by_hand = featurize.generate_feature_matrix_by_hand(data, degree=i+1)
+        by_sklearn = generate_feature_matrix(data, degree=i+1)
```

Comment:
line 9 has an error. should read `featurize.generate_feature_matrix(data, degree=i+1)` instead of `generate_feature_matrix(data, degree=i+1)`. This seams to be the source of the import error that the CI is running into.

Reply:
Fixed.